src/Umbraco.Core/Services/FileServiceOperationBase.cs

```diff
@@ -82,6 +82,11 @@ protected async Task<TOperationStatus> HandleDeleteAsync(string path, Guid userK
 
     protected async Task<Attempt<TEntity?, TOperationStatus>> HandleCreateAsync(string name, string? parentPath, string? content, Guid userKey)
     {
+        if (name.Contains("/"))
+        {
+            return Attempt.FailWithStatus<TEntity?, TOperationStatus>(InvalidName, default);
+        }
```

Comment:
I reckon we need this both here and in `HandleRenameAsync`?

Reply:
Done 😁